overlay.yaml

```diff
@@ -230,7 +230,18 @@ actions:
   - target: $["components"]["schemas"]["RequestConfigInputUpdate"]
     update:
       "x-speakeasy-plan-validators": "RequestConfigInputValidator"
-
+  
   # Don't attempt to unmarshal 404 respones
   - target: $["paths"]["/appstore/requestable_permissions/{permission_id}"]["get"]["responses"]["404"]["content"]
-    remove: true
\ No newline at end of file
+    remove: true
+
+  # Handle FastAPI vs. Starlette HTTPException diffs -
+  #  https://fastapi.tiangolo.com/tutorial/handling-errors/#fastapis-httpexception-vs-starlettes-httpexception
+  - target: $["components"]["schemas"]["HTTPValidationError"]["properties"]["detail"]
+    update:
+      "oneOf": [{"type": "array", "items": {"$ref": "#/components/schemas/ValidationError"}}, {"type": "string"}]
+  - target: $["components"]["schemas"]["HTTPValidationError"]["properties"]["detail"]["type"]
+    remove: true
+  - target: $["components"]["schemas"]["HTTPValidationError"]["properties"]["detail"]["items"]
```

Comment:
This new data in the overlay was generated by creating a revision of `original-openapi.json` with my expected `oneOf` schema changes.

Then ran `speakeasy overlay compare` against `original-openapi.json` and my revised spec.

This is the overlay that Speakeasy spit out for that!